### PR TITLE
port tut plugin to sbt 1.0.x

### PR DESCRIPTION
**Pull request by larsrh**
Second part of #173

As discussed with @tpolecat, drop support for Scala 2.10 and sbt 0.13.x.

### CODE REVIEW

**Comment by larsrh** [comment]
@tpolecat GitHub will show the contents of #174 in this pull request too, because you forked off the branch before #174 was merged to `master`.

**Comment by tpolecat** [comment]
Ok this looks good. There's no way to implement the watch test?

**Comment by larsrh** [comment]
Unless you're fine with hooking into the `sbt` package: sbt/io#67.

**Comment by tpolecat** [comment]
Ah ok, I see. No, that's fine. We can add it back if they fix that.

**Comment by larsrh** [comment]
Something really strange is going on in Travis – no idea why it has just run one build instead of two?

**Comment by tpolecat** [comment]
I don't know what the matrix thing does, I don't recall when that was added.

**Comment by tpolecat** [comment]
I deleted the caches. Can't hurt.

**Comment by larsrh** [comment]
Let me try this again.

**Comment by larsrh** [comment]
Okay, I ran out of time today. I believe the solution is to migrate the scripted tests fully to 2.12. I will do that at a later time.

**Comment by tpolecat** [comment]
👍  Thanks so much for your help with this.

**Comment by larsrh** [comment]
I think I have figured out the `sbt-scripted` problem. If Travis is green, you can merge this.

File: core/build.sbt [hunk 1]
```diff
@@ -4,18 +4,15 @@ libraryDependencies ++= Seq(
...
-      libraryDependencies.value :+ "org.scala-lang.modules" %% "scala-xml" % "1.0.5"
-    case _ =>
-      libraryDependencies.value
-  }
-}
+libraryDependencies += "org.scala-lang.modules" %% "scala-xml" % "1.0.5"
+
+// scripted-plugin is enabled by default, in particular in this non-sbt subproject
+// this means that switching to 2.11.11 will result in non-existent dependencies
+libraryDependencies ~= { _.filterNot(_.organization == "org.scala-sbt") }
```
Comment (larsrh) [inline core/build.sbt]: @dwijnand This is probably of interest to you. I'm not sure whether it was a good design decision to make the scripted plugin enabled by default. (But having an sbt plugin and non-sbt code in the same build is an edge case, so I'm not sure it's that sever.)

Comment (larsrh) [inline core/build.sbt]: I was speaking too early. This workaround actually breaks execution of scripted tests.

Comment (tpolecat) [inline core/build.sbt]: So close! I am in awe of your tenacity.

Comment (larsrh) [inline core/build.sbt]: I'm a grad student. I'm being paid to be tenacious.

**Comment by larsrh** [comment]
All green now. :shipit:

**Comment by tpolecat** [comment]
🔥

**Comment by tpolecat** [comment]
I will attempt a release today.

**Comment by larsrh** [comment]
Before you do, I'll send you another one forward-porting #168.

**Comment by tpolecat** [comment]
👍
